examples/data-objects/clicker/src/agent.ts

```diff
@@ -3,13 +3,16 @@
  * Licensed under the MIT License.
  */
 
-import { IFluidRouter, IFluidRunnable, IRequest, IResponse } from "@fluidframework/core-interfaces";
+import { IFluidRunnable, IRequest, IResponse } from "@fluidframework/core-interfaces";
 import { SharedCounter } from "@fluidframework/counter";
 
 // Sample agent to run.
-export class ClickerAgent implements IFluidRouter, IFluidRunnable {
+export class ClickerAgent implements IFluidRunnable {
 	constructor(private readonly counter: SharedCounter) {}
 
+	/**
```

Comment:
For this (or anything else) that isn't exposed publicly on a published package, you don't need to deprecate and should instead just remove or make whatever code updates are required to fix.  This will include all examples.

E.g. here you can just remove the router bits.

Reply:
Agreed that this doesn't need to be deprecated. I wanted to mark all the places that are using this pattern so we can remove it all at once.

Reply:
An old branch where I had worked on replacing request with entryPoint in one of our examples, in case it's useful: https://github.com/microsoft/FluidFramework/compare/main...alexvy86:FluidFramework:usage-examples

Reply:
TBH I would do the example updates in main immediately/separately from this PR

Reply:
https://github.com/microsoft/FluidFramework/pull/16575